server/routes/journeys/dailySchedule/handlers/downloadCsvHandler.ts

```diff
@@ -47,9 +47,22 @@ export default class DownloadCsvHandler implements PageHandler {
         'Appointment subtype': a.appointmentSubtypeDescription || '',
         'Room location': a.appointmentLocationDescription,
         'Court or probation team': a.externalAgencyDescription || '',
-        'Video link': a.videoLink || '',
+        'Video link': this.toFullCourtLink(a) || '',
         'Last updated': formatDate(a.lastUpdatedOrCreated, "d MMMM yyyy 'at' HH:mm"),
       })),
     )
   }
+
+  private toFullCourtLink = (item: ScheduleItem) => {
+    if (item.videoLinkRequired) {
+      if (item.videoLink) {
+        return item.videoLink
+      }
+      if (item.hmctsNumber) {
+        return `HMCTS${item.hmctsNumber}@meet.video.justice.gov.uk`
```

Comment:
Yes I realise this is hardcoded and not config based.

Reply:
I will at least centralise this in my next piece of work as it is used in the templates as well, so now in two places.  

Also needed for printing next.